packages/date-picker/src/vaadin-infinite-scroller.js

```diff
@@ -18,15 +18,11 @@ template.innerHTML = `
     }
 
     #scroller {
-      position: relative;
```

Comment:
Let's try to not move styles around like with this `position: relative` to minimize diff.